app/connectors/CalculationDetailsConnector.scala

```diff
@@ -28,7 +28,7 @@ class CalculationDetailsConnector @Inject()(httpClient: HttpClient,
                                            (implicit ec: ExecutionContext) {
 
   def getCalculationDetails(nino: String, calculationId: String)(implicit hc: HeaderCarrier): Future[CalculationDetailResponse] = {
-    val getCalculationDetailsUrl: String = appConfig.desBaseUrl + s"/income-tax/view/calculations/liability/$nino/$calculationId"
+    val getCalculationDetailsUrl: String = appConfig.ifBaseUrl + s"/income-tax/view/calculations/liability/$nino/$calculationId"
```

Comment:
if these connector calls should be IF calls now I think you'll need to make a new trait like IFConnector so that the IF bearer token is passed along.

Reply:
you could copy https://github.com/hmrc/income-tax-employment/blob/main/app/connectors/IFConnector.scala and update to fetch the config you have added for IF.